packages/terra-form-textarea/src/TextareaField.jsx

```diff
@@ -112,21 +132,26 @@ const defaultProps = {
   maxWidth: undefined,
   onChange: undefined,
   onInput: undefined,
+  rows: null,
```

Comment:
`null` and `undefined` have slightly different meanings. This should either be unspecified/unlisted from default props or `undefined`

Reply:
Setting default value to undefined made similar change in textarea also ca966a9.